cpp/src/rolling/rolling.cu

```diff
@@ -255,9 +256,17 @@ gdf_column* rolling_window(const gdf_column &input_col,
   // Use the column wrapper class from io/utilities to quickly create a column
   gdf_column_wrapper output_col(input_col.size,
                                 input_col.dtype,
-                                gdf_dtype_extra_info{TIME_UNIT_NONE},
+                                input_col.dtype_info,
                                 "");
 
+  if (input_col.col_name != nullptr) {
```

Comment:
This will be done by gdf_column_view_augmented when you pass a string name to gdf_column_wrapper constructor. So just change the creation of output_col above.